HeterogeneousCore/CUDATest/test/BuildFile.xml

```diff
@@ -0,0 +1,12 @@
+<iftool name="cuda-gcc-support">
+<bin file="test*.cc" name="testHeterogeneousCoreCUDATest">
+  <use name="FWCore/TestProcessor"/>
+  <use name="HeterogeneousCore/CUDACore"/>
+  <use name="catch2"/>
+</bin>
+
+<bin file="TestCUDATest.cc">
+  <flags TEST_RUNNER_ARGS=" /bin/bash HeterogeneousCore/CUDATest/test runtests.sh"/>
```

Comment:
scram does now support running scripts directly without having to use the test runner. See
https://github.com/cms-sw/cmssw/blob/master/FWCore/Framework/test/BuildFile.xml#L332

Reply:
Thanks, I've missed that update.